deploy/examples/create-external-cluster-resources.py

```diff
@@ -250,6 +252,10 @@ def validate_rgw_metadata_ec_pool_name(self):
                     "Provided rbd_data_pool name, {}, does not exist".format(rbd_pool_name))
             return rbd_metadata_ec_pool_name
 
+    def dry_run(self, msg):
+        if self._arg_parser.dry_run:
+            print("Execute: " + "'" + msg+"'")
```

Comment:
this would be the correct indent, print("Execute: " + "'" + msg + "'")

You can use `Pylint` for linter